app/src/main/java/com/example/marvel/data/api/ComicsService.kt

```diff
@@ -14,6 +14,6 @@ interface ComicsService {
 
 
     @GET("/v1/public/comics/{id}?ts=1&apikey=c41a380d16444cae4dfa217d408336e1&hash=78abe8add9743df8eff80d42a5672cdf")
```

Comment:
Imagine você quer ter duas chaves, uma que é usada pra desenvolvimento e outra pra fazer o release. Se tiver hardcoded desse jeito você teria dificuldades. O ideal seria criar uma config no gradle contendo esses caras =)

https://developer.android.com/studio/build/gradle-tips#share-custom-fields-and-resource-values-with-your-app-code

Além disso essa `apiKey` e `hash` vão ser usadas em todos os requests então vai ter bastante duplicação de código. Minha sugestão seria criar um Interceptor que vai adicionar esses campos em todos os requests.

https://square.github.io/okhttp/interceptors/

Reply:
Massa!!
A próxima tarefa é para integrar o fluxo de detalhes com a API da Marvel. Vou aplicar essa sugestão nessa próxima tarefa, acho q faz mais sentido ;)